### DISCUSSION TITLE
How to start theme config wizard

### DISCUSSION THREAD

**Discussion by cmonty14** [ohmybash/oh-my-bash#611] [Q&A]
Hi,
I want to use theme _powerbash10k_ and set `OSH_THEME="powerbash10k"` in my ~/.bashrc accordingly.

Question:
How can I start theme wizard to configure it?

THX

**Comment by akinomyoga** [comment]
This is *not* powerlevel10k. There is no configuration wizard. Contributions to add a wizard are welcome.

**Comment by cmonty14** [comment]
This means any theme in oh-my-bash is "as-is" and not customizable?

**Reply by akinomyoga** [reply]
It depends on the theme. Some parts can be configurable through the variables prepared in `~/.bashrc`. Some themes provide additional configurations, which may be described in the respective README or in the source code of the theme.

**Reply by akinomyoga** [reply]
The configuration of `powerbash10k` is described [here](https://github.com/ohmybash/oh-my-bash/tree/master/themes/powerbash10k#configuration).
